common/persistence/persistenceMetricClients.go

```diff
@@ -379,6 +379,8 @@ func (p *workflowExecutionPersistenceClient) updateErrorMetric(scope int, err er
 	case *TimeoutError:
 		p.metricClient.IncCounter(scope, metrics.PersistenceErrTimeoutCounter)
 		p.metricClient.IncCounter(scope, metrics.PersistenceFailures)
+	case *workflow.LimitExceededError:
```

Comment:
LimitExceededError is application level error.  I don't think we return this error anywhere in the persistence code.  Use ServerBusy for any throttling.

Reply:
sure